opentracing-kafka-spring/src/test/java/io/opentracing/contrib/kafka/spring/Listener.java

```diff
@@ -26,7 +26,6 @@
 
   @KafkaListener(topics = "spring")
   public void listen(String message) {
-    // TODO: no active span
     System.out.println("active span: " + tracer.activeSpan());
```

Comment:
@orange-buffalo could you add a  check that active span is not null here anymore?

Reply:
Sure, added an assertion.